### PR TITLE
allow paging to be disabled

### PR DESCRIPTION
**Pull request by tdamsma**
See https://github.com/Pegase745/sqlalchemy-datatables/issues/56

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -14,6 +14,7 @@ Added
   - Reimplement API to rely on SQLALchemy core functions.
   - Return error messages as 'error' property in result.
   - Add tearDown to the tests.
+  - support 'length=-1' to disable paging
```
Comment (Pegase745) [inline CHANGELOG.rst]: It's a keepachangelog format, so:
```
Unreleased
~~~~~~~~~
Added
~~~~~
  - Support `length=-1` to disable paging.
```

with the commit message always beginning with a capital letter, and an imperative verb and ending with a period, please.

Comment (tdamsma) [inline CHANGELOG.rst]: Sure, I'll update it right away

**Comment by Pegase745** [comment]
You kept it under the section 1.0.0. This commit is part of a future release known as being `Unreleased` for now.

Source [keepachangelog.com](keepachangelog.com)

```
Always have an "Unreleased" section at the top for keeping track of any changes.

This serves two purposes:

People can see what changes they might expect in upcoming releases
At release time, you just have to change "Unreleased" to the version number and add a new "Unreleased" header at the top.
```

**Comment by tdamsma** [comment]
Oops, I clearly wasn't paying attention. Didn't even see you released 1.0.0, congrats!

**Comment by Pegase745** [comment]
@tdamsma it's thanks to you !

**Comment by tdamsma** [comment]
Any idea what's wrong with coveralls? nothing on http://status.coveralls.io/

**Pull request merged**